research-software-directory/RSD-as-a-service
# 182 more data surfconext

# Collect more data from SURFconext

Changes proposed in this pull request:

* Collect name, email and organisation from SURFconext
* Upgrade dependencies and add a logging dependency
* Some small code improvements

How to test:

* This change affects the database structure, so we need to rebuild everything.
* `docker-compose down --volumes`
* `docker-compose build`
* `docker-compose up --scale scrapers=0`
* Login to SURFconext, this should be successful
* Connect to the database, and check that the query `SELECT * FROM login_for_account` contains values for name, email and home_organisation

Closes #182

PR Checklist:
